tensorrt_llm/llmapi/mpi_session.py

```diff
@@ -318,10 +345,9 @@ def __init__(self,
                  is_comm: bool = False):
         # FIXME: this is a hack to avoid circular import, resolve later
         from tensorrt_llm.executor.ipc import ZeroMqQueue
-        self.addr = addr
         self.queue = ZeroMqQueue((addr, hmac_key),
-                                 is_server=True,
-                                 use_hmac_encryption=bool(hmac_key))
+                                 use_hmac_encryption=bool(hmac_key),
+                                 is_server=False)
```

Comment:
What's the reason of switching the server/client concept here?

Its kind of confusing to me.